packages/dashboard/src/components/queryEditor/helpers/getCorrectSelectionMode.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { DashboardWidget } from '~/types';
+import { getCorrectSelectionMode } from './getCorrectSelectionMode';
+
+const createMockWidget = (widgetType: string): DashboardWidget => {
+  return {
+    type: widgetType,
+    id: 'test-id',
+    x: 0,
+    y: 0,
+    z: 0,
+    height: 20,
+    width: 20,
+    properties: {},
+  };
+};
+
+describe('Get correct selection mode', () => {
```

Comment:
[Nit]: Pass function directly to describe to avoid typing string and keeping in sync with function. 

```ts
describe(getCorrectSelectionMode, () => {});
```